extensions/3rdparty/LyricWiki/Tag_Lyric.php

```diff
@@ -66,19 +65,10 @@
 	$wgHooks['AfterViewUpdates'][] = "efApplyIndexPolicy";
 }
 
-#Install extension
-function lyricTag()
-{
-	// Keep track of whether this is the first <lyric> tag on the page - this is to prevent too many Ringtones ad links.
-	global $wgFirstLyricTag;
-	$wgFirstLyricTag = true;
-}
-
 function lyricTag_InstallParser( $parser ) {
 	#install hook on the element <lyric>
 	$parser->setHook("lyric", "renderLyricTag");
 	$parser->setHook("lyrics", "renderLyricTag");
-	$parser->setHook("tonefuze", "renderToneFuzeTag");
```

Comment:
FWIW: I did poke around my devbox to find templates that used this, then updated those templates in production on LyricWiki at that time. Hopefully we'll be good, but the community should catch it pretty quickly if we failed.